### ISSUE TITLE
Feature Request: Set frame for camera

### ISSUE THREAD

**Issue by rcheetah** [BlenderHQ/multiple_camera_render#4]
Blender seems to make it really hard to have different states of a scene. One way to achieve that, is to make adjustments in frames (given you don't want to render a sequence to start with). Could there maybe a setup to let Multiple Camera Render render different cameras on different frames? Maybe with a custom property of the camera which specifies which frame should be rendered.

My current workflow is, to store my cameras in separate collections called "F1", "F2", "F3" and so on. I can then set the frame and render all cameras of that given frame. But it would be event better, if the script could render *all* the cameras and figure out which frame to render by itself. 

I'm sorry if this request is somehow dumb or irrelevant. I don't work with blender on a regular basis, but I sometimes need if for some projects. If there is a better way to achieve this, than my current workflow, please tell me. Thank you very much!

**Comment by ivan-perevala** [comment]
Hi, @rcheetah !

---

The current implementation was created for a specific task - the creation of synthetic datasets for photogrammetric software. And the workflow for which it was designed did not foresee that the number of cameras would change between frames. It works like this: first, the queue of cameras that need to be rendered is determined. Then, when changing frames, the camera changes to the next one from this queue. This is actually a brief description of how the add-on currently works.

I am currently busy with the development of the Camera Projection Painter addon, so I have very little time for others at the moment, but yes, I noticed that both @ssh4net and other users ask about specific work tasks that come down to rendering sequentially or different objects, or different cameras. To be honest, I don't think that it is necessary to write an addon for each of these tasks, it can mostly be solved with a simple script for batch data processing.

---

If you have suggestions for a specific solution that will not destroy the current workflow with this add-on, but only complement it, I will try to implement it. If so, please describe in more detail what your workflow should look like, and how to link it to the current one.

**Comment by rcheetah** [comment]
Hi @ivan-perevala, thanks for your reply!

I see! I didn't realize the specific use case for this addon (mostly because I don't know enough about photogrammetry, and I think I just blanked out when reading the addon description). Thank you anyway, because it really eases my workflow, even if I use it for things it wasn't made for. 

I understand, that this might not be the ideal plugin for the problem I described, and any "solution" of course shouldn't interfere  with the original purpose of the plugin. Thank you though for the hint with the scripts. I will look in to that (I have not yet used scripts in blender). Maybe I can even write a plugin myself. We’ll see. 

Thank you for your kind reply!
